src/lib.rs

```diff
@@ -538,6 +586,92 @@ impl PublicKey {
         })
     }
 
+     // Partially Blind a message to be signed with public metadata
+     pub fn blind_with_metadata<R: CryptoRng + RngCore>(
+        &self,
+        rng: &mut R,
+        msg: impl AsRef<[u8]>,
+        metadata: &[u8],
```

Comment:
why not `impl AsRef<..>` here, too?

Reply:
done, updated all instances of metadata to match this